defaults/main.yml

```
@@ -26,6 +26,9 @@ galaxy_zergpool_socket_name: zergpool.sock
 # Where the socket for the zergling will be placed
 galaxy_zergpool_listen_path: "{{ galaxy_mutable_data_dir }}/{{ galaxy_zergpool_socket_name }}"
 
+galaxy_systemd_zergling_uwsgi_config_style: "{{ galaxy_config_style | default('ini')}}"
+galaxy_systemd_zergling_uwsgi_config_file: "{{ galaxy_config_dir }}/uwsgi.ini"
```

```suggestion
galaxy_systemd_zergling_uwsgi_config_file: "{{ galaxy_config_file }}"
```

this will pull the value from ansible-galaxy, probably better than defaulting to ini